app/api/email_utils.py

```diff
@@ -137,6 +137,48 @@ def send_email_mentorship_relation_accepted(request_id):
     )
     send_email(request_sender.email, subject, html)
 
+def send_email_report_violation(user_id, comment_id):
+    """
+    Sends an email to the admins as a notification,
+    for a violation reported by a user on a task comment.
+
+    Args:
+        user_id: ID of user reporting the violation
+        comment_id: ID of comment that is being reported
+    """
+
+    from app.database.models.user import UserModel
+    from app.database.models.task_comment import TaskCommentModel
+
+    # User who's reporting the violation
+    user = UserModel.find_by_id(user_id)
+    # Get the name of the user
+    user_name = user.name
+    # Email of user
+    user_email = user.email
+
+    # Get task comment
+    task_comment = TaskCommentModel.find_by_id(comment_id)
+    # Comment string
+    comment = task_comment.comment
+
+    # Commentor of the task comment being reported
+    commenter = UserModel.find_by_id(task_comment.user_id)
+    # Commentor name
+    commenter_name = commenter.name
+    # Commentor email
+    commenter_email = commenter.email
+
+    subject = "Violation Reported"
+    html = render_template(
+        "email_report_violation.html",
+        reporter_name=user_name,
+        reporter_email=user_email,
+        comment=comment,
+        commenter_name=commenter_name,
+        commenter_email=commenter_email
+    )
+    send_email("opensource@anitab.org", subject, html)
```

Comment:
By sending to admins I meant to the emails, of the admins. So imagine, that we have 2 users (Maria and Joana) in the system which are admins of the application, I think the idea of this was to send to admins email. So you would fetch the admins using functions you have available, and send this to both Maria and Joana. This is what I thought this issue was created for

Reply:
Done